src/main/java/core/basesyntax/App.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class App {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure randomFigure = figureSupplier.getRandomFigure();
+        int num = 7;
+        for (int i = 1; i < num + 1; i++) {
+            if (i <= ((num + 1) / 2)) {
+                System.out.println(figureSupplier.getRandomFigure());
```

Comment:
you already saved it into local variable?

Reply:
I don't know what you mean. it feels that I miss something, but I just do not get it.